src/plugins/auto/multi_schedule.cpp

```diff
@@ -154,8 +165,19 @@ void MultiSchedule::GenerateWorkers(const std::string& device,
                 IdleGuard<NotBusyWorkerRequests> idleGuard{workerRequestPtr, *idleWorkerRequestsPtr};
                 workerRequestPtr->_exceptionPtr = exceptionPtr;
                 {
-                    auto capturedTask = std::move(workerRequestPtr->_task);
-                    capturedTask();
+                    if (workerRequestPtr->_exceptionPtr != nullptr) {
+                        if (workerRequestPtr->_reload) {
```

Comment:
multi will be merged with auto logic soon, probably we can leave multi unchanged for now, just focus on auto

Reply:
Okay.